### PR TITLE
A context manager to suppress unwanted prints

### PR DESCRIPTION
**Pull request by dansterenson**
## Description
Added a context manager which gives the ability to temporarily suppress console output and used it in the update release notes manager when executing the validator. It was necessary because the validator would print warnings about packages that we do not necessarily want to update.

https://stackoverflow.com/questions/6735917/redirecting-stdout-to-nothing-in-python

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by reutshal — CHANGES_REQUESTED** [review]
Nice

File: demisto_sdk/commands/common/tests/tools_test.py [hunk 1]
```diff
@@ -1112,3 +1112,27 @@ def test_get_release_note_entries(version, expected_result):
     """
 
     assert get_release_note_entries(version) == expected_result
+
+
+def test_suppress_stdout(capsys):
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply demisto_sdk/commands/common/tests/tools_test.py:1130]: Done

File: demisto_sdk/commands/common/tools.py [hunk 2]
```diff
@@ -1945,3 +1946,23 @@ def get_current_tags() -> list:
...
+    """
+        Temporarily suppress console output without effecting error outputs.
+        Example of use:
+
+            with suppress_stdout():
+                print('This message will not be printed')
+            print('This message will be printed')
+    """
+    with open(os.devnull, "w") as devnull:
+        old_stdout = sys.stdout
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply demisto_sdk/commands/common/tools.py]: Done

**Review by reutshal — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,7 @@
 * Fixed an issue where **openapi-codegen** entered into an infinite loop on circular references in the swagger file.
 * The **format** command will now set `fromVersion: 6.2.0` for widgets with 'metrics' data type.
 * Updated the **find-dependencies** command to support generic modules, definitions, fields and types.
+* Added a function which gives the ability to temporarily suppress console output.
```
Comment (reutshal) [inline CHANGELOG.md:10]:
```suggestion
* Added a tool that gives the ability to temporarily suppress console output.
```

Comment (dansterenson) [reply CHANGELOG.md:10]: Done

**Pull request merged**